composer.json

```diff
@@ -52,6 +52,9 @@
 		"check-cs": [
 			"\"vendor/bin/phpcs\""
 		],
+		"check-cs-errors": [
+			"\"vendor/bin/phpcs\" --error-severity=1 --warning-severity=6"
```

Comment:
I added --warning-severity=6 because default warnings seem to be triggered at `5`